tests/opentracer/test_tracer_gevent.py

```diff
@@ -2,17 +2,17 @@
 from opentracing.scope_managers.gevent import GeventScopeManager
 import pytest
 
-import ddtrace
-from ddtrace.contrib.gevent import patch
-from ddtrace.contrib.gevent import unpatch
+from ddtrace.contrib.gevent import context_provider
```

Comment:
```suggestion
from ddtrace.contrib.internal.gevent import context_provider
```

Reply:
`context_provider` is only defined in  `ddtrace.contrib.gevent.__init__.py`. We expose this to customers but honestly we don't need to do this anymore. The gevent contextprovider is deprecated no longer used. We can replace all references to `ddtrace.contrib.gevent.context_provider` with the default context provider (tracer.context_provider).